symplyphysics/laws/thermodynamics/maxwell_boltzmann_statistics/speed_distribution.py

```diff
@@ -41,6 +54,59 @@
     * exp(-1 * particle_mass * particle_speed**2 / (2 * units.boltzmann_constant * equilibrium_temperature))
 )
 
+# Derive from Maxwell-Boltzmann distribution of velocity vector components
+
+_velocity_component_distribution = velocity_component_distribution.law.rhs.subs({
+    velocity_component_distribution.particle_mass: particle_mass,
+    velocity_component_distribution.equilibrium_temperature: equilibrium_temperature,
+})
+
+_velocity_x, _velocity_y, _velocity_z = sym_symbols("velocity_x:z", real=True)
+
+_velocity_x_distribution = _velocity_component_distribution.subs(
+    velocity_component_distribution.velocity_component, _velocity_x
+)
+
+_velocity_y_distribution = _velocity_component_distribution.subs(
+    velocity_component_distribution.velocity_component, _velocity_y
+)
+
+_velocity_z_distribution = _velocity_component_distribution.subs(
+    velocity_component_distribution.velocity_component, _velocity_z
+)
+
+_spherical_coordinate_system = CoordinateSystem(CoordinateSystem.System.SPHERICAL)
+
+_radius, _azimuthal_angle, _polar_angle = _spherical_coordinate_system.coord_system.base_scalars()
+
+# The speed distribution depends solely on the radial component of the velocity vector.
+# Therefore we integrate over polar and azimuthal angles to get rid of them.
+_spherical_volume_element = (
```

Comment:
Maybe `_spherical_volume_velocity_element` ?
It will justify the replacement of radius to speed.